pages/models.py

```diff
@@ -137,6 +137,8 @@ def purge_fastly_cache(sender, instance, **kwargs):
     Requires settings.FASTLY_API_KEY being set
     """
     purge_url(f'/{instance.path}')
+    if not instance.path.endswith('/'):
```

Comment:
do we need to do this in the job models, too?

Reply:
Those urls are already properly resolved with a trailing slash, see

https://github.com/python/pythondotorg/blob/main/jobs/urls.py#L27

for example, which is purged via https://github.com/python/pythondotorg/blob/main/jobs/models.py#L274.